### PR TITLE
Clean up tooltips for consistency

### LINKED ISSUE #308: Improve consistency of CLI tooltips
Package [cmd](https://github.com/ubclaunchpad/inertia/tree/master/cmd) contains Inertia's CLI command declarations - each command has several properties that help document what they do:

```go
var cmdAddRemote = &cobra.Command{
	Use:   "add [REMOTE]",
	Short: "Add a reference to a remote VPS instance",
	Long: `Add a reference to a remote VPS instance. Requires 
information about the VPS including IP address, user and a PEM
file. Specify a VPS name.`,
        // ...
```

Some issues to address:
* grammatical errors/inconsistencies, I think there are a few. This includes the use of periods (`.`), etc - I think for `Short`, exclude the period, and for `Long`, include a period
* argument indicators: the Cobra library uses `[argument]` as a standard, so all references to `[ARGUMENT]` should be lowercased to follow that standard

### PR DESCRIPTION
**Pull request by bobheadxi**
:tickets: **Ticket(s)**: Closes #308

---

## :construction_worker: Changes

Some tweaks to the wording, punctuation, casing, etc of the tooltips to better align with Cobra (the CLI library) standards

## :flashlight: Testing Instructions

```bash
make cli
inertia # poke around commands
```

### CODE REVIEW

File: cmd/deploy.go [hunk 1]
```diff
@@ -51,25 +51,23 @@ func init() {
...
+			Use:    remote.Name + " [command]",
 			Hidden: true,
 			Short:  "Configure deployment to " + remote.Name,
 			Long: `Manage deployment on specified remote.
 
 Requires:
-1. an Inertia daemon running on your remote - use 'inertia [REMOTE] init'
-   to set one up.
-2. a deploy key to be registered for the daemon with your GitHub repository.
+1. an Inertia daemon running on your remote - use 'inertia [REMOTE] init' to get it running.
```
Comment (PiggySpeed) [inline cmd/deploy.go]: `[REMOTE]` should now be lowercase `[remote]`

File: cmd/deploy.go [hunk 2]
```diff
@@ -51,25 +51,23 @@ func init() {
...
+2. a deploy key to be registered within your remote repository for the daemon to use.
 
-Continuous deployment requires a webhook url to registered for the daemon
-with your GitHub repository.
+Continuous deployment requires the daemon's webhook address to be registered in your remtoe repository.
```
Comment (leesw98) [inline cmd/deploy.go]: `remtoe`

File: cmd/deploy.go (hunk 1 above)
Comment (bobheadxi) [reply cmd/deploy.go]: 🔥 6121b81

File: cmd/deploy.go [hunk 3]
```diff
@@ -359,7 +357,7 @@ var cmdDeploymentPrune = &cobra.Command{
 var cmdDeploymentSSH = &cobra.Command{
 	Use:   "ssh",
 	Short: "Start an interactive SSH session",
-	Long:  `Starts up an interact SSH session with your remote.`,
+	Long:  `Starts an interact SSH session with your remote.`,
```
Comment (leesw98) [inline cmd/deploy.go]: Stay consistent with `Start` or `Starts`

Comment (bobheadxi) [reply cmd/deploy.go]: ur wrong

Comment (leesw98) [reply cmd/deploy.go]: Lol

**Pull request merged**